trackintel/preprocessing/positionfixes.py

```diff
@@ -326,3 +366,49 @@ def extract_triplegs(positionfixes, staypoints=None, *args, **kwargs):
     ret_triplegs['id'] = ret_triplegs['id'].astype('int')
 
     return ret_triplegs
+
+
+def propagate_tripleg(pfs, stp, position_edge_posfix_tl, direction=1):
```

Comment:
Don't remember if we have a convention for internal functions (start with _?; is this even an internal function?), but otherwise a small docstring could be helpful, esp. if this function turns up in the (automatically generated) documentation at some point.